<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>David A Knowles</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
		<link rel="stylesheet" href="assets/css/main.css" />
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		
		<style>
			.image.face {
						float: left;
						margin: 0 0.5em 0.5em 0;
						top: 0.25em;
						width: 150;
					}
			</style>
	</head>
	<body>

		<!-- Header -->
			<section id="header">
				<header>
					<span class="image avatar"><img src="images/logo.png" alt="" /></span>
					<h1 id="logo"><a href="#">Knowles lab</a></h1>
						<p>Machine learning and genomics</p>
				</header>
				<nav id="nav">
					<ul>
						<!-- <li><a href="#one" class="active">About</a></li> -->
						<li><a href="#two">People</a></li>
						<li><a href="#three">Publications</a></li>
						<li><a href="#four">Software</a></li>
						<li><a href="#five">Reports/Abstracts/Presentations</a></li>
						<li><a href="#six">Miscellaneous</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://twitter.com/david_a_knowles" class="icon fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://github.com/davidaknowles/" class="icon fa-github"><span class="label">Github</span></a></li>
						<li><a href="mailto:david.arthur.knowles@gmail.com" class="icon fa-envelope"><span class="label">Email</span></a></li>
					</ul>
				</footer>
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">

						<!-- One -->
							<section id="one">
								<div class="container">
									<header class="major">
										<h2>Knowles lab</h2>
					<p><a href="http://www.nygenome.org/">New York Genome Center</a> / Columbia University</p>
									</header>
									<p>The Knowles lab develops and applies machine learning methods for data analysis challenges in genomics. We're particularly interested in understanding the role of transcriptomic dysregulation across the spectrum from rare to common genetic disease. This involves better characterization of the genetic and environmental factors contributing to mRNA expression and splicing variation. We collaborate with diverse research groups at NYGC and beyond collecting large-scale genomics datasets in the context of <a href="http://www.nygenome.org/lab-groups-overview/center-genomics-neurodegenerative-disease-phatnani-lab/">neurological disease</a> and developing <a href="http://www.nygenome.org/lab-groups-overview/technology-innovation-lab/">novel genomic technologies</a> including <a href="https://satijalab.org/">single cell methods</a>, <a href="http://sanjanalab.org/home.html">forward genetic screens</a> and <a href="https://tllab.org/">long-read transcriptomics</a>.</p>

									<p>If you're interested in machine-learning-flavored genomics, gene regulation and/or RNA splicing, please get in <a href="mailto:daknowles@nygenome.org">touch</a>! We have experimental lab space so that includes wet lab people looking to do damp (mixed computational/experimental) projects. I'm also interested in hearing from machine learning folks who are curious about applying their expertise in genetics/genomics.</p>
									
									<p>All lab members will have the option of being cross-trained in both computational analysis and experimental work. Postdocs and students are eligible for an appointment at Columbia University.</p>									<ul class="feature-icons">
										<li class="fa-scissors"><a href="#splicing">RNA splicing</a></li>
										<li class="fa-fighter-jet"><a href="#eagle">Gene-by-environment</a></li>
										<!-- <li class="fa-microchip">Digital histology</li> -->
										<li class="fa-wrench"><a href="#vb">Variational methods</a></li>
										<li class="fa-snowflake-o"><a href="#bnp">Bayesian nonparametrics</a></li>
									</ul>
									
									<section id="splicing">
										<div>
									<h4>RNA splicing</h4>
									<p>We developed <a href="https://github.com/davidaknowles/leafcutter">LeafCutter</a> to identify, quantify and test variable intron splicing events, obviating the need for accurate transcript annotations and circumventing the challenges in estimating relative isoform abundance (<a href="#LeafCutter">link</a>). An early version of <em>LeafCutter</em> was used in <a href="#Li2016splicing">our study</a> linking complex disease and splicing. </p>
								</div>
							</section>
								<section id="eagle"> <div>
								<h4>Genetic differences in environmental response</h4>
								<p>Phenotypic variation results from the interplay of genotype and environment: in particular genetic variation can modulate the transcriptomic response to perturbation. However, detecting gene by environment effects on the transcriptome is challenging in observational data. We developed a computational method, <a href="#Knowles2017gxe">EAGLE</a>, which leverages allele-specific expression as a controlled, with-in individual test of the influence of environment factors on different genetic backgrounds. Code is available on <a href="https://github.com/davidaknowles/eagle">github</a>. </p>
								<p>Direct in vitro perturbation of cells can greatly increase power to detect GxE interaction effects due to increased effect sizes and controlled conditions. We have used human iPSC-derived somatic  cells to understand the genetic basis of cardiac sensitvity to doxorubicin. Anthracycline-cardiotoxicity (ACT) is a common side-effect of chemotherapy, but its genetic basis and molecular mechanism remain unclear. We measured transcriptomic and cellular damage response to doxorubicin in a panel of iPSC-derived cardiomyocytes from 45 individuals. Using a novel, efficient linear mixed model, <a href="https://github.com/davidaknowles/suez">suez</a>, we identified hundreds of loci that interact with ACT response. These response-QTLs are significantly enriched in ACT GWAS (<a href="#Knowles2018dox">link</a>). </p>
								 </div> </section>
								 <section id="vb"> <div>
								<h4>Variational methods</h4>
								<p>Variational methods offer a computationally efficient alternative to Markov chain Monte Carlo algorithms for inference in Bayesian probabilistic models. My work allows such methods to be more easily applied to a broader range of probabilistic models. I extended variational message passing to "non-conjugate" (intuitively, less tractable) models, and incorporated this method, <a href="#Knowles2011c">Non-conjugate VMP</a> (NCVMP), into the publicly available <a href="http://infernet.azurewebsites.net/">Infer.NET</a> software package. Later, Tim Salimans and I did early work on <a href='#salimans2014using'>using Monte Carlo estimation within variational learning</a>, which is now an active subfield of research under the moniker "Stochastic Variational Inference".</p>
</div> </section>
							<section id="bnp"> <div>
							<h4>Bayesian nonparametric models</h4>
							<p>Bayesian nonparametric (BNP) models are a category of statistical methods that automatically adapt their complexity to observed data. I have developed BNP methods for <a href="#Knowles2014">hierarchical clustering</a>, <a href="#wilson2011gaussian">heteroskedastic multivariate regression</a>, <a #href>network data</a>, <a href="#Palla2012nonparametric">variable clustering</a> and <a href='#Knowles2011nonparametric'>nonparametric sparse factor analysis</a> (NSFA). In particular, NSFA is able to adaptively choose an appropriate number of factors from data. I used this method to delineate gene co-expression modules from microarray data, and other researchers have subsequently used it in diverse applications including image denoising and EEG analysis.</p>
</div> </section>
								

								</div>
							</section>

						<!-- Two -->
							<section id="two">
								<div class="container">
									<h2>People</h2>
              <section id="david"> <div>
								<h4>David A. Knowles, Principal Investigator</h4>
							<p><div><span class="image face"><img src="images/avatar.jpg" alt="Profile pic" width=150/></span></div>I was a post-doctoral researcher at Stanford University with <a href="http://med.stanford.edu/plevritis.html">Sylvia Plevritis</a> (Center for Computational Systems Biology/Radiology) and <a href="http://pritchardlab.stanford.edu/">Jonathan Pritchard</a> (Genetics) having previously worked with <a href="http://ai.stanford.edu/~koller/">Daphne Koller</a> prior to her move to Coursera. I did my PhD with <a href="http://learning.eng.cam.ac.uk/zoubin/">Zoubin Ghahramani</a> in the <a href="http://mlg.eng.cam.ac.uk/">Machine Learning</a> group of the Cambridge University Engineering Department. I was the Roger Needham Scholar at Wolfson College, funded by Microsoft Research. My undergraduate degree comprised two years of Physics before switching to Engineering to complete an MEng with Zoubin. I took the <a href="http://www.bioinformatics.ic.ac.uk/">MSc Bioinformatics and Systems Biology</a> at <a href="http://www3.imperial.ac.uk/">Imperial College</a> in 2007/8. You can download my <a href="documents/david-knowles-resume.pdf">CV</a> </p> 
							</div> </section>
						
						<section id="staff"> <div>
						 	<h3>Staff</h3>
							<p></p>

							<section id="laura"> <div>
								<h4>Laura Pereira - Staff Scientist/Lab Manager</h4>
							<p><div><span class="image face"><img src="images/laura.jpg" alt="Laura" width=150/></span></div>Laura joined the New York Genome Center as a Staff Scientist and Lab Manager for David’s lab. She is working on determining what is the contribution of genomic sequences and RNA-binding proteins in regulating splicing in different cellular contexts with a focus on neuronal function. In the lab, we use in vitro differentiation models of human embryonic stem cells together with CRISPR/Cas systems to understand how splicing decisions are undertaken at a molecular level.</p>
							<p>Laura has a B.S in Biochemistry from the Universidad de la Republica, Uruguay. She obtained her PhD at the University of Illinois at Chicago studying the transcriptional networks involved in regulating embryonic stem cell self-renewal and differentiation. For her postdoctoral training, Laura joined Oliver Hobert’s Lab at Columbia University. As a postdoc she studied what signals are required to induce and maintain neuronal identity to give rise to the adult sexually dimorphic nervous system.</p>
							<p>Outside of work, Laura enjoys playing volleyball, biking on the Hudson and spending time with her family at the many NYC playgrounds and museums.</p> 
			  </div> </section>
			  
							</div> </section>
							
						<section id="postdocs"> <div>
						 	<h3>Postdocs</h3>
			  
			  <section id="megan"> <div>
				<h4>Megan Schertzer</h4>
			<p><div><span class="image face"><img src="images/megan.jpg" alt="Megan" width=150/></span></div><p>Megan joined the Knowles Lab as a postdoctoral research associate in 2019. She is interested in integrating human genomics and molecular biology to better understand the role of non-coding RNA in the brain. She originally completed a BS in biochemistry from Lee University in Tennessee, and obtained her PhD under the mentorship of Mauro Calabrese at UNC-Chapel Hill. There, she led a project to study how cis-acting long non-coding RNAs efficiently target and spread Polycomb Repressive Complexes to compact chromatin and repress nearby gene expression. Outside of the lab, she enjoys hiking, playing volleyball, and eating Mexican food.</p> 
</div> </section>

<section id="brielin"> <div>
	<h4>Brielin C Brown</h4>
			  <p><div><span class="image face"><img src="images/brielin.jpg" alt="Brielin" width=150/></span></div><p>Brielin is a research fellow at the Data Science Institute, jointly affiliated with the New York Genome Center. Brielin's research lies at the intersection of machine learning and genomics. He is broadly interested in understanding how genetics and environment combine to lead to disease through changes in cellular function. To this end, he develops machine learning algorithms for modeling and inference in large-scale genomic studies. Before coming to Columbia, Brielin completed a PhD in Computer Science from the University of California, Berkeley and worked as a computational biologist at Verily Life Sciences. As a PhD student, Brielin was supported by an NSF fellowship and Chancellors fellowship for graduate study. He completed his undergraduate studies at the University of Virginia, earning a BS in Physics and a BA in Computer Science. In his spare time, Brielin enjoys techno music and is an avid surfer.</p> 
			</div> </section>
							
						<section id="phds"> <div>
               <h3>PhD students</h3>
               
               <section id="andrew"> <div>
								<h4>Andrew Stirn</h4>
							<p><div><span class="image face"><img src="images/andrew.jpg" alt="Andrew" width=150/></span></div>Andrew is a PhD student in the Computer Science Program at Columbia University, where he recently completed his Master’s degree. He is broadly interested in developing deep variational Bayesian inference techniques with an eye for biological applications. Prior to Columbia, he was designing sensors, embedded systems, and algorithms for consumer wearable technologies and has taken several products to market. During this time, he developed a passion for research and machine learning that compelled his return to graduate school. He received a BSc in Electrical Engineering from Johns Hopkins University in 2008. Andrew is an avid telemark skier. When there is no snowy mountain to be found, he is on his bike preparing for when there will be.</p> 
			  </div> </section>
			  
			</div> </section> 
			
						
			<section id="mscs"> <div>
				<h3>Masters students</h3>
				
				<section id="stephen"> <div>
								 <h4>Stephen Malina</h4>
							 <p><div><span class="image face"><img src="images/stephen.jpg" alt="Stephen" width=150/></span></div>Stephen is a Master's student in the Computer Science Program at Columbia. He's interested in machine learning methods that can assist with mechanistic understanding of biology. Before joining Columbia, he worked as a backend software engineer, which prepared him for his current work with machine learning and biology by teaching him that everything's always more complicated than expected.</p> <br> 
			   </div> </section>

			   <section id="nasrine"> <div>
				<h4>Nasrine Metic</h4>
			<p><div><span class="image face"><img src="images/nasrine.png" alt="Nasrine" width=150/></span></div>A creative and curious thinker, Nasrine is a master graduate student at Ecole Polytechnique Federale de Lausanne, Switzerland. She is completing her Master's degree in the Bioengineering Program along with a Minor in Biocomputing. During her studies and internships, she developed a keen interest in computational biology: she would like to focus on leveraging machine learning algorithms for computational biology to help understand genetically-linked diseases.</p> <br> 
			</div> </section>
			   
			   
			 </div> </section> 

			 						
			<section id="undergrads"> <div>
				<h3>Undergraduate students</h3>
				
				<section id="collin"> <div>
								 <h4>Collin Wang</h4>
							 <p><div><span class="image face"><img src="images/collin.jpg" alt="Collin" width=150/></span></div>Collin is a first year computer science major at Columbia University with an interest in developing and applying novel machine learning methods to understanding the behavior of alternative splicing and RNA  binding proteins. Previously, he has worked on developing models to understand the behavior of transcription factors and non-coding genetic regulatory variants. In his free time, Collin enjoys running, snowboarding, reading, and listening to country music.</p> 
			   </div> </section>

			   <section id="Udai"><div>
				<h4>Udai Nagpal</h4>
				<p><div><span class="image face"><img src="images/udai.jpg" alt="Udai" width=150/></span></div>Udai is an undergraduate student at Columbia Engineering studying Computer Science. He is interested in applied machine learning and probabilistic modeling, and is currently exploring active learning approaches with applications in computational biology. In his free time, Udai enjoys skiing and playing tennis.
				</p> 
			</div> </section>
			   
			 </div> </section> 

              <section id="join"> <div>
                <h3>Join us!</h3>
							<p>We're always looking for great candidates to join the lab at all levels. If you're a student at Columbia University feel free to <a href="mailto:david.arthur.knowles@gmail.com">e-mail</a> me about rotating in the lab. If you're looking at PhD programmes please bear in mind you will need to apply through Columbia University <a href="https://www.cs.columbia.edu/">Computer Science</a> or <a href="https://systemsbiology.columbia.edu/">Systems Biology</a>.</p>
							</div> </section>

							</div> </section>

						<!-- Three -->
							<section id="three">
								<div class="container">
									<h3>Publications</h3>
									<ul class="feature-icons">
										<li class="fa-bullhorn"><a href="#working">Working</a></li>
										<li class="fa-flask"><a href="#genetics">Genetics</a></li>
										<li class="fa-cog"><a href="#ml">Machine learning/statistics</a></li>
									</ul>